### DISCUSSION TITLE
Tesseract530 - tesseract.exe Flagged by Bitdefender as infected with Gen:Variant.Fragtor.64431

### DISCUSSION THREAD

**Discussion by kstrife7** [SubtitleEdit/subtitleedit#7008] [General]
As the title states, Tesseract530 - tesseract.exe Flagged by Bitdefender as infected with Gen:Variant.Fragtor.64431.

Need confirmation that this is a false positive. 

Been using SubtitleEdit and Tesseract 5.3.0 for weeks now without issues, suddenly today Bitdefender is going nuts with this and going on about how it's infected with Gen:Variant.Fragtor.64431, which is usually a pretty generic message.  

Just looking for confirmation that this is a false positive.

**Comment by niksedk** [comment]
This is a common problem with releasing software... false positives. You can try to report it to your anti virus provider, and see what they say.

I've updated Tesseract 5 to the latest 5.3.1 in latest SE beta version (portable version);
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip

Tesseract versions included with SE are from here: https://github.com/UB-Mannheim/tesseract/wiki
